integration/test_filesystems_integration.py

```diff
@@ -246,3 +249,56 @@ def test_write_to_parquet_to_v3io_with_indices(setup_teardown_test):
 
     read_back_df = pd.read_parquet(out_file, columns=columns)
     assert read_back_df.equals(expected), f"{read_back_df}\n!=\n{expected}"
+
+
+def test_write_to_tsdb():
+    table_name = f'tsdb_path-{int(time.time_ns() / 1000)}'
+    tsdb_path = f'v3io://bigdata/{table_name}'
```

Comment:
We have a pattern for this using uuid, e.g.:
https://github.com/mlrun/storey/blob/70b5186bfd05ac10767cf56afa0224308b9cdbe6/tests/test_flow.py#L1462

Reply:
Ah, it was there before too... You just moved it...